src/models/backupTarget.js

```
@@ -0,0 +1,116 @@
+            payload: location.pathname.startsWith('/backupTarget') ? queryString.parse(location.search) : {},
+          })
+        }
+      })
+    },
+  },
+  effects: {
+    *query({
+      payload,
+    }, { call, put }) {
+      const data = yield call(queryBackupTarget, payload)
+      if (payload && payload.field && payload.keyword && data.data) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)